proposals/0176-scc-parsing.rst

```diff
@@ -3,8 +3,8 @@ Meaning-preserving parsing rules for SCC annotations
 
 .. author:: Vladislav Zavialov
 .. date-accepted:: 2019-04-17
-.. ticket-url::
-.. implemented::
+.. ticket-url:: https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0176-scc-parsing.rst
```

Comment:
The ticket url should be something in the gitlab (either an issue or a MR)

Reply:
Fixed.